aspnetcore/tutorials/grpc/grpc-start.md

```diff
@@ -130,11 +122,11 @@ info: Microsoft.Hosting.Lifetime[0]
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
-For Visual Studio for Mac, see the .NET 7 version of this tutorial.
-
-<!--
-Follow the instructions in [Building a complete .NET Core solution on macOS using Visual Studio for Mac](/dotnet/core/tutorials/using-on-mac-vs-full-solution) to create a console app with the name *GrpcGreeterClient*.
--->
+* In Visual Studio 2022 for Mac and select **File** > **Add** > **Project...**.
+* In the **Choose a template for your new project** dialog, select **Web and Console** > **App** > **Console Application**, and select **Continue**.
+* Select **.NET 8.0** for the target framework and select **Continue**.
+* Name the project **GrpcGreeterClient**. It's important to name the project *GrpcGreeter* so the namespaces match when you copy and paste code.
```

Comment:
Is the correct project name GrpcGreeterClient or GrpcGreeter?

Reply:
@tdykstra, ouch, sorry, three bad copy/paste errors in row!  It is supposed to be *GrpcGreeterClient* for the namespace and the client project name.  Thanks.